tests/unit/helpers/validate-properties-fail-test.js

```diff
@@ -0,0 +1,61 @@
+import { moduleFor } from 'ember-qunit';
```

Comment:
@bcardarella We would have to have a different module for these tests because we modify `deepEqual` in the `beforeEach`